src/extractor/edge_based_graph_factory.cpp

```diff
@@ -394,6 +395,52 @@ void EdgeBasedGraphFactory::GenerateEdgeExpandedEdges(
     util::DEBUG_TURNS_START(debug_turns_path);
 #endif
 
+    const auto getRepresentativeCoordinate = [this](const NodeID src, const NodeID tgt,
```

Comment:
Will clear this up in the process. Currently I am more in a Beta kind of stage. Have to take another approach to reasonably get correct results. But I will make sure to get reliable names in there.